js/components/foia_personnel.jsx

```diff
@@ -18,15 +18,15 @@ function FoiaPersonnel({ foiaPersonnel }) {
   const email = (foiaPersonnel.email || '').toLowerCase();
   return (
     <div>
-      <p className="submission-help_poc">{ displayName(foiaPersonnel) }</p>
+      <div className="submission-help_poc">{ displayName(foiaPersonnel) }</div>
       {
         (foiaPersonnel.phone || [])
-          .map(phone => <p className="submission-help_phone" key={phone} >{ phone }</p>)
+          .map(phone => <div className="submission-help_phone" key={phone}>{ phone }</div>)
       }
       { foiaPersonnel.email &&
-        <p className="submission-help_email">
+        <div className="submission-help_email">
```

Comment:
What's the reason for changing all these to `div`s? Seems less semantic?

Reply:
1. the margin on `p` give it too much spacing. 2, its not really copy, its data so i was thinking div is less semantic but p would be the wrong semantic 🤷‍♀️